server/speak.py

```diff
@@ -1,12 +1,14 @@
 # speak.py
 from gtts import gTTS
-from playsound import playsound
-
-FILENAME = "tts.mp3"
+import os
+import utils
 
 def get_speak (text, lang):
-    print("speak" + text)
-    print("speaklang" + lang)
+    filename = utils.random_tts_filename()
+    utils.delete_all_tts_files()
     tts = gTTS(text=text, lang=lang, slow=False)
-    tts.save(FILENAME)
-    playsound(FILENAME)
\ No newline at end of file
+    tts.save(filename)
+    return filename
+    
+
+mp3_path = os.path.join(os.path.dirname(__file__), "../static/dist/FILENAME")
```

Comment:
This line is no longer needed, right ?

Reply:
Yeah, it makes no sense, the variable is assigned at app.py and we can handle it there.